src/components/VideoPlayer/VideoInformation/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useContext } from 'react';
+import Grid from '@material-ui/core/Grid';
+import AddToFavorites from '../AddToFavorites';
+import { store } from '../../../state/store';
+
+import { Title, Description } from './style';
+import Profile from '../Profile';
+
+const VideoInformation = () => {
+  const globalState = useContext(store);
+  const title =
+    globalState.state.currentVideo.snippet === undefined
+      ? ''
+      : globalState.state.currentVideo.snippet.title;
```

Comment:
Maybe this would be better
```suggestion
      globalState?.state?.currentVideo?.snippet?.title;
```